saleor/static/scss/layouts/_cart.scss

```diff
@@ -58,6 +58,10 @@
       svg {
         vertical-align: text-top;
       }
+      svg:hover path{
```

Comment:
This change seems unrelated to the issue being solved in this PR. I'd rather remove it and consider in a separate ticket if we should highlight the SVG icons.